### PR TITLE
initial tradfri blind quirks

### PR DESCRIPTION
**Pull request by billyburly**

### CODE REVIEW

File: zhaquirks/ikea/opencloseremote.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                    Ota.cluster_id,
+                    WindowCovering.cluster_id,
+                    LightLink.cluster_id,
+                ],
+            }
+        },
+    }
+
+    replacement = {
+        MODELS_INFO: [(IKEA, "TRADFRI open/close remote")],
```
Comment (dmulcahey) [inline zhaquirks/ikea/opencloseremote.py]: you can remove this

File: zhaquirks/ikea/opencloseremote.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+)
+
+IKEA_CLUSTER_ID = 0xFC7C  # decimal = 64636
+
+
+class IkeaTradfriOpenCloseRemote(CustomDevice):
+    """Custom device representing IKEA of Sweden TRADFRI remote control."""
+
+    signature = {
+        MODELS_INFO: [("\x02KE", "TRADFRI open/close remote")],
```
Comment (dmulcahey) [inline zhaquirks/ikea/opencloseremote.py]: Wait, is it actually reported as "\x02KE"

**Comment by dmulcahey** [comment]
@billyburly any chance you can finish this up?

**Comment by dmulcahey** [comment]
bump... maybe??

File: zhaquirks/ikea/opencloseremote.py (hunk 2 above)
Comment (jmoransalama) [reply zhaquirks/ikea/opencloseremote.py:36]: This is the info for my remote:
```
  "manufacturer": "\u0002KE",
  "model": "TRADFRI open/close remote"
```
Hope it helps!

**Comment by dmulcahey** [comment]
@billyburly are you still interested in getting this merged?

**Comment by dmulcahey** [comment]
@billyburly I fixed the sorting and formatting. Thanks for the contribution.

**Pull request merged**